src/annotation/Validator.java

```diff
@@ -0,0 +1,52 @@
+package annotation;
+
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Validator {
+
+    public static List<?> validate(Object object) throws IllegalAccessException {
+        List<String> list = new ArrayList<>();
+        Class<?> objectClass = object.getClass();
+
+        Field[] fields = objectClass.getDeclaredFields();
+        for (Field field : fields) {
+            field.setAccessible(true);
+            Object fieldValue = field.get(object);
+            if (field.isAnnotationPresent(Length.class)) {
+                String currentValue = (String)fieldValue;
+                if (currentValue.length() < field.getAnnotation(Length.class).min() || currentValue.length() > field.getAnnotation(Length.class).max() || currentValue =="") {
```

Comment:
Strings and Boxed types should be compared using "equals()"

It’s almost always a mistake to compare two instances of java.lang.String or boxed types like java.lang.Integer using reference equality == or !=, because it is not comparing actual value but locations in memory.